Fix vale warnings (#2075)
codenvy/che-docs: modules/installation-guide/partials/proc_manually-upgrading-che-in-operatorhub.adoc

// upgrading-che-using-operatorhub

[id="manually-upgrading-che-in-operatorhub_{context}"]
= Manually upgrading {prod-short} in OperatorHub

OperatorHub is an assembly point for sharing Operators. The OperatorHub helps you deploy and update applications. The following section describes the process of upgrading {prod-short} by using OperatorHub and the `Manual` approval strategy approach. Use the `Manual` approval strategy to prevent automatic updates of the Operator with every release.

.Prerequisites

* An administrator account on an instance of OpenShift.

> * An instance of an earlier minor version of {prod-short}. You have installed the instance by using the Operator.

Reviewer: `from OperatorHub` is important

Reviewer 2: ```suggestion
* An instance of an earlier minor version of {prod-short} that you installed by using OperatorHub.
```
IBM SG:
```
that conjunction
Use "that", without a comma, to introduce a restrictive clause. Use "which", preceded by a comma, to introduce a
nonrestrictive clause.
Consider the following examples: (1) The system units, which have two drives, are floor-standing models. (Nonrestrictive) (2)
The system units that have two drives are floor-standing models. (Restrictive) In the first example, all of the system units are
floor-standing models and each has two drives. The second example implies that some of the system units have two drives
and some do not; the sentence refers only to system units with two drives. See the IBM Style topics Commas and Pronouns for
more information.
```

Reviewer 2: If the reader isn't the person who installed the "earlier minor version of {prod-short}," how can they determine whether it was installed by using OperatorHub versus some other method?

Reviewer 3: @rolfedh 
What about:

1) An existing earlier instance of {prod-short} installed from OperatorHub. 

2) An existing instance of {prod-short} {prod-prev-ver} or older, installed by OperatorHub.

cc @tolusha

Reviewer 2: How about: "An instance of {prod-short} {prod-prev-ver} or earlier that was installed by using OperatorHub."
- We can drop "existing" as redundant.
- IBM SG recommends using "earlier" or "later" to describe versions.

Reviewer 2: IBM SG
```
References to earlier or later versions
To state a requirement that includes earlier or later versions of a product, use the phrases or earlier to indicate earlier versions
and or later to indicate subsequent versions.
To use the transformers, you need DB2 Universal Database 5.1 or later.
This memo applies to IBM Mobile Connect 3.2 or earlier.
Install DB2 Database for Linux, UNIX, and Windows 9.7 or later.
```

Author: @tolusha @rolfedh @MichalMaler it's fixed. Thank you so much for your suggestions!

[id="manually-upgrading-che-in-operatorhub_{context}"]

= Manually upgrading {prod-short} in OperatorHub

> This section describes how to upgrade {prod-short} by using the OperatorHub with the `Manual` approval strategy.

Reviewer 2: IBM SG
```
Avoid self-referential text whenever possible, such as "This topic is about" or "This section describes". Instead, focus on
the purpose or value of the content, and include words that users are likely to search for. For example, in a short
description in an online help topic, provide users with enough information to determine whether the topic meets their
needs. You can use self-referential text in learning modules and tutorials.
```

Reviewer 2: IBM SG
```
Second person
As much as possible, use the second person, which is indicated by the personal pronoun you. The second person focuses on the reader.
> When you create a database, you must provide a unique name.
```

.Prerequisites

* An administrator account on an instance of OpenShift.

* An instance of an earlier minor version of {prod-short}. You have installed the instance by using the Operator.

* The approval strategy in the subscription is `Manual`.

.Procedure

. Open the OpenShift web console.

. Navigate to the *Operators* -> *Installed Operators* page.

. In the list of the installed Operators, click *{prod}*.

. Navigate to the *Subscription* tab. 

. Next to the *Upgrade Status*, inspect the upgrades that require approval. The expected message is *1 requires approval*.

. Click *1 requires approval*.

. Click *Preview Install Plan*.

. Review the resources that are available for upgrade and click *Approve*.

.Verification steps

. Navigate to the *Operators* -> *Installed Operators* page to monitor the upgrade progress. When complete, the status changes to *Succeeded* and *Up to date*.

> . The {prod-ver} version number is visible at the end of the page.

Reviewer 2: Is there a way to make this step a user action, such as "Verify that the version number at the end of the page is correct." Or, consider combining this step with the result in the previous step, "When complete, the status changes to *Succeeded* and *Up to date*, and the end of the page displays the version number."
